### ISSUE TITLE
multisigAddress is inconsistent with the one generated by NEO-GUI

### ISSUE THREAD

**Issue by cloud8little** [neow3j/neow3j#7]
I have three WIF, and generate a 2-signed address, expected it be Adkvh59aCdeEeidXCzAkAyoqYsSBpSvRPw
` ECKeyPair ecKeyPair1 = ECKeyPair.create(WIF.getPrivateKeyFromWIF("L3XMvv21K4LUZdZkUaxcm7WTpkaE1FqhyMKXmvc7gkfmjhxP4UFu"));
        ECKeyPair ecKeyPair2 = ECKeyPair.create(WIF.getPrivateKeyFromWIF("L4ZCr2DsZe5nAkzMA5AVwMpjHnRrq11VoNdxgx7NQxJgip9n6nds"));
        ECKeyPair ecKeyPair3 = ECKeyPair.create(WIF.getPrivateKeyFromWIF("L53KB4Jmvk5GiVC1RX7GxNRb7b73Aiq9EthfjcQk6TznT6sCyXqy"));
        String multiSigAddress = Keys.getMultiSigAddress(2, new BigInteger[]{ecKeyPair1.getPublicKey(), ecKeyPair2.getPublicKey(), ecKeyPair3.getPublicKey()});`

please have a  check. Thanks.

**Comment by gsmachado** [comment]
Hey @cloud8little!

Thanks for the issue.

So, I'm not sure if this is a bug on neow3j, OR if this is a "bug" (or malfunction) from the `neo-python` and `neo-vm` implementations.

Based on the code found [here](https://github.com/CityOfZion/neo-python/blob/f570ce99302033b6276e338ee5043449b118ba70/neo/SmartContract/Contract.py#L104) and [here](https://github.com/neo-project/neo/blob/ecb9b15d0cc1241ed8cca4853de50de33861e4c0/neo/SmartContract/Contract.cs#L65), it seems that NEO does a "sorting" of public keys to finally build the verification script for a multisig address.

However, my questions are: 
- Sorting the public keys is a good/right thing to do?
- Or should we keep order, as in other blockchain projects?

**Comment by gsmachado** [comment]
Ah, just to clear it out:

If you switch the public key's order of `Keys.getMultiSigAddress()` method, it would display the `Adkvh59aCdeEeidXCzAkAyoqYsSBpSvRPw` address.

Instead of:

```
String multiSigAddress = Keys.getMultiSigAddress(2, ecKeyPair1.getPublicKey(), ecKeyPair2.getPublicKey(), ecKeyPair3.getPublicKey());
```

switch the order to:

```
String multiSigAddress = Keys.getMultiSigAddress(2, ecKeyPair3.getPublicKey(), ecKeyPair1.getPublicKey(), ecKeyPair2.getPublicKey());
```

However, as I said in my previous comment, this is debatable.

**Comment by cloud8little** [comment]
@gsmachado Tried with `String multiSigAddress = Keys.getMultiSigAddress(2, ecKeyPair3.getPublicKey(), 
ecKeyPair1.getPublicKey(), ecKeyPair2.getPublicKey());` and the result is right at this time. Could you solve this problem through neow3j at this moment? since I don't have contact with City-of-Zion team, so let's try to solve this in neow3j first.

**Comment by gsmachado** [comment]
@cloud8little: I posted a question on the #python channel in Discord (to the maintainers of `neo-python`, and CoZ team).

The question I posted [here](https://github.com/neow3j/neow3j/issues/7#issuecomment-450510665) goes beyond whether `neow3j` should be fixed or not. 
Actually, it's a design question and `neow3j` can follow it or not. The point is that it seems `neo` code and `neo-python` are sorting the signatures instead of respecting the order which they were specified. I think this is not standardized in NEO, but I need to first discuss (clarify) that with someone before changing anything.
